Spokestack/TextToSpeech.swift

```diff
@@ -343,33 +357,27 @@ private let apiQueue = DispatchQueue(label: TTSSpeechQueueName, qos: .userInitia
 
 // MARK: Internal data structures
 
-fileprivate struct TTSResponseURL: Codable {
-    let url: URL
-}
-
-// SSML
-fileprivate struct TTSSSMLResponseSynthesize: Codable {
-    let synthesizeSsml: TTSResponseURL
+fileprivate struct TTSResponseErrorLocation: Codable {
```

Comment:
Do you mean to tell me that Java actually requires _less_ boilerplate here? I left this deserialization out entirely because I'll never report these.

Reply:
Don't get me started. Just know that this is even less obtuse than the older approaches.